frontend/src/__tests__/cypress/cypress/tests/mocked/modelRegistry/modelVersionDeploy.cy.ts

```diff
@@ -341,4 +341,63 @@ describe('Deploy model version', () => {
     kserveModal.selectExistingConnectionSelectOptionByResourceName('test-secret-not-match');
     kserveModal.findExistingConnectionSelect().should('not.contain.text', 'Recommended');
   });
+
+  it('Test project dropdown scroll in Deploy model modal', () => {
```

Comment:
this test passes even without the `isScrollable` added. i don't think it is necessary anyway as you end up just testing a patternfly component